signals/parser/api.py

```diff
@@ -33,6 +33,15 @@ def process_authorization_attribute(self, attribute):
             error_message = "Found invalid authorization attribute: {} for {}, exiting."
             raise SignalsError(error_message.format(attribute, self.url_path))
 
+    # def validate(self, endpoint_json):
```

Comment:
Yeah, just didn't want to delete those until I got the OK to leave my code in the schema.py file.